cmd/ssl-expiry-check/main.go

```diff
@@ -15,19 +15,31 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"os"
 	"strconv"
+	"time"
 
 	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/checkclient"
+	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/nodeCheck"
 	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/ssl_util"
+	"github.com/Comcast/kuberhealthy/v2/pkg/kubeClient"
 	log "github.com/sirupsen/logrus"
 )
 
-var domainName string
-var portNum string
-var daysToExpire string
-var insecureCheck string
+const defaultCheckTimeout = 10 * time.Minute
+
+var (
+	domainName    string
+	portNum       string
+	daysToExpire  string
+	insecureCheck string
+	insecureBool  bool
+)
+
+// CheckTimeout is a variable for how long code should run before it should retry.
+var CheckTimeout time.Duration
```

Comment:
i don't think you need to make this a global var -- seems like its only used in `init()`?

Reply:
This is for the enhancement in the comment above, for using the `GetDeadline` func. Removed until I have that working.